main/EditText/index.tsx

```diff
@@ -198,15 +199,11 @@ function EditText(props: Props): ReactElement {
             autoCapitalize={autoCapitalize}
             onFocus={(): void => {
               setFocus(true);
-              if (onFocus) {
-                onFocus();
-              }
+              onFocus && onFocus();
```

Comment:
How about using optional chaining? 
```
onFocus?.();
```

Reply:
Good idea. However, we need to upgrade the [typescript](https://github.com/dooboolab/dooboo-ui/blob/fa61f04b7789236ceb8d3957f9da82541d430dc3/package.json#L63) beforehand.

Reply:
I got it. Let's leave it for the next.